dockerfiles/Dockerfile.healthcheck

```diff
@@ -0,0 +1,2 @@
+FROM ubuntu:20.04
```

Comment:
Nit: the above one is ubuntu:22.04 this one is 20.04

Reply:
fixed! tks